dnf-behave-tests/features/microdnf/module-reset.feature

```diff
@@ -0,0 +1,143 @@
+      | Module | State | Stream | Profiles |
+      | nodejs |       |        |          |
+ When I execute dnf with args "module list nodejs"
+  Then module list contains
+      | Repository             | Name   | Stream | Profiles                          |
+      | microdnf-module-enable | nodejs | 8 [d]  | default [d], development, minimal |
+      | microdnf-module-enable | nodejs | 10     | default [d], development, minimal |
+      | microdnf-module-enable | nodejs | 11     | default, development, minimal     |
+
+
+Scenario: I can reset a disabled non-default stream back to a non-default state
+ When I execute dnf with args "module disable dwm"
+ Then the exit code is 0
+  And modules state is following
+      | Module | State    | Stream | Profiles |
+      | dwm    | disabled |        |          |
+ When I execute dnf with args "module list"
+ Then module list contains
+      | Repository             | Name | Stream   | Profiles |
+      | microdnf-module-enable | dwm  | 6.0 [x]  | default  |
+ When I execute dnf with args "module reset dwm"
+ Then the exit code is 0
+  And stdout contains "Resetting modules:"
+  And stdout contains "dwm"
+  And modules state is following
+      | Module | State | Stream | Profiles |
+      | dwm    |       |        |          |
+ When I execute dnf with args "module list"
+ Then module list contains
+      | Repository             | Name | Stream | Profiles |
+      | microdnf-module-enable | dwm  | 6.0    | default  |
+
+
+Scenario: Resetting of a default stream does nothing
+ When I execute dnf with args "module list nodejs"
+ Then module list contains
+      | Repository             | Name   | Stream | Profiles                          |
+      | microdnf-module-enable | nodejs | 8 [d]  | default [d], development, minimal |
+      | microdnf-module-enable | nodejs | 10     | default [d], development, minimal |
+      | microdnf-module-enable | nodejs | 11     | default, development, minimal     |
+ When I execute dnf with args "module reset nodejs"
+ Then the exit code is 0
+  And stdout contains "Nothing to do"
+  And modules state is following
+      | Module | State | Stream | Profiles |
+      | nodejs |       |        |          |
+ When I execute dnf with args "module list nodejs"
+ Then module list contains
+      | Repository             | Name   | Stream | Profiles                          |
+      | microdnf-module-enable | nodejs | 8 [d]  | default [d], development, minimal |
+      | microdnf-module-enable | nodejs | 10     | default [d], development, minimal |
+      | microdnf-module-enable | nodejs | 11     | default, development, minimal     |
+
+
+Scenario: Resetting of a non-default non-enabled stream does nothing
+ When I execute dnf with args "module list dwm"
+ Then module list contains
+      | Repository             | Name | Stream | Profiles |
+      | microdnf-module-enable | dwm  | 6.0    | default  |
+ Then I execute dnf with args "module reset dwm"
```

Comment:
`microdnf`?

Reply:
I think I lost one commit while squashing them locally. it was the find->replace commit probably :/